ios/Exponent/Kernel/Services/EXScreenOrientationManager.m

```diff
@@ -35,7 +35,7 @@ - (UIInterfaceOrientationMask)supportedInterfaceOrientationsForVisibleApp
 - (void)handleScreenOrientationChange:(UITraitCollection *)traitCollection
 {
   for(NSString *experienceId in _subscribedModules) {
-    EXScreenOrientation *subscribedModule = [_subscribedModules objectForKey:experienceId];
+    id<EXScreenOrientationListener> subscribedModule = [_subscribedModules objectForKey:experienceId];
```

Comment:
```suggestion
    id<EXScreenOrientationListener> subscribedModule = _subscribedModules[experienceId];
```
if we change this line…

Reply:
Then it says:
Expected method to read dictionary element not found on object of type 'NSMapTable<NSString *,id> * _Nonnull'